### ISSUE TITLE
[batterymonitor@pdcurtis] Persistent notification about "User difined alarm sound selection" !

### ISSUE THREAD

**Issue by Nokia808** [linuxmint/cinnamon-spices-applets#2511]
```
 * Applet version/Build date: 1.3.8
 * Cinnamon version: 4.2.2
 * Distribution: Fedora 30 X 64 bit
 * Graphics hardware *and* driver used: Hybrid VGA (Intel Corporation HD Graphic 5500 + Radeon R7 M265 2GB)
 * Snippet or attachment of ~/.cinnamon/glass.log
 * Snippet or attachment of ~/.xsession-errors

 ```

**Notify author of applet**
@pdcurtis 

**Issue**
I received new version of this applet. I tried to investigate it's setting if something changed. I saw sound alarm for low battery level was disabled though I'm already enabled it since very long time (on old versions). So, I suspect that this is due to new version update. Then I enabled it. After enabling this option, Cinnamon started to display a notification about that EVERY time launch or restart Cinnamon !!

**Expected behaviour**

Cinnamon should not display this notification every time restart or launch Cinnamon ! It should display it for only one time, just after enable sound alarm for low battery level ....

**Other information**

Please see attached screenshot (notice, kindly, notification).

![screenshot](https://user-images.githubusercontent.com/25375330/62416666-979e5e00-b62e-11e9-8de8-27ef62949482.png)

**Comment by pdcurtis** [comment]
@Nokia808 This is not an error, it is a feature and it is quite deliberate that one is reminded every start up. 

The only point in an audible alert is to attract attention when one is away from the machine otherwise the modal alert stops you doing anything. It has to be a sound that is loud, long and attention grabbing. The one I use personally is a sound file I found for which is used somewhere for emergency evacuation of a building which cuts through most background and the effect in a quiet library, workplace or train at full volume could cause some concern. This is actually discussed in help file.

What I am considering is putting an additional checkbox in the settings to inhibit this notification for those who do not need it the next time I do an update.

In the meantime if you really have a real problem with this you could comment out the line in `.local/share/cinnamon/applets/batterymonitor@pdcurtis/3.2/applet.js` at about  line 171 starting `Main.warningNotify`

Can you also please close this as an issue or change it to a [Feature Request]

**Comment by Nokia808** [comment]
@pdcurtis 
I agree with your plan for future version to add new option to mute this notification if user wish to this.
I will wait your next version.
Thank you for your care !

**Comment by pdcurtis** [comment]
@Nokia808 i think this addresses your concerns and also enables a wider range of audio file formats to be used.

Can you please check on your machine  - on mine the enhanced widget selects .mp3 files but the play button does not work for .mp3 but does for .oga . It seems to be a Cinnamon issue as the enhancement is new and is only available on 4.2 so I may be the first user.

**Comment by Nokia808** [comment]
@pdcurtis 
Hi. I'm very sorry for the delay in my replay ! I did not see your last comment unless few minutes ago !

I test new feature "Provide notification when user specified sound file is in use". It is working.

I will close this issue. However, I'm still seeing that add a check box within notification itself when appear, checking it lead to disable such notification, is still nice feature making your applet more brilliant.

Thanks!
